test/sampler_test.cc

```diff
@@ -53,13 +53,13 @@ TEST_F(SamplerTest, sample_sgd_single_variable) {
   sampler->set_random_seed(1, 1, 1);
 
   sampler->sample_sgd_single_variable(0, 0.1);
-  EXPECT_EQ(infrs->weight_values[0], 0.1);
+  EXPECT_EQ(infrs->weight_values[0], 0.2);
```

Comment:
The default step size is `0.01`, and I assume that was at least somewhat arbitrary... The default decay is 0.95, so after about 13 epochs, `0.02` would become `0.01` anyway...